SimplenoteUITests/NoteEditor.swift

```diff
@@ -23,8 +23,20 @@ class NoteEditor {
         app.textViews.element.clearAndEnterText(text: "")
     }
 
-    class func clearAndEnterText(enteredValue: String) {
-        app.textViews.element.clearAndEnterText(text: enteredValue)
+    class func clearAndEnterText(enteredValue: String, viaPaste: Bool = false) {
+        let noteContentTextView = app.textViews.element
+        guard noteContentTextView.exists else { return }
+
+        if viaPaste {
+            UIPasteboard.general.strings = []
```

Comment:
While this theoretically is not needed, there was some flakiness (like using old content for a new note) unless the buffer was cleared.

Reply:
Added a comment, thanks!